src/SprykerEco/Zed/Adyen/Business/Handler/Notification/AdyenNotificationHandler.php

```diff
@@ -93,6 +93,14 @@ protected function handleNotification(AdyenNotificationRequestItemTransfer $noti
             $this->utilEncodingService->encodeJson($notificationTransfer->getAdditionalData())
         );
 
+        if ($notificationTransfer->getEventCode() === $this->config->getAdyenNotificationEventCodeAuthorisation()) {
```

Comment:
- Extract this condition to the method with a meaningful name.
- `isDuplicateAuthorisationNotification()`.

Reply:
extracted code to a method